### ISSUE TITLE
Unable to run StratisD on Testnet

### ISSUE THREAD

**Issue by ranjancse26** [stratisproject/StratisBitcoinFullNode#4179]
I am having issues in running the latest code of StratisD on testnet.

Unable to resolve service for type 'Stratis.SmartContracts.Core.Receipts.IReceiptRepository' while attempting to activate 'Stratis.Bitcoin.Features.SmartContracts.ReflectionExecutor.Controllers.SwaggerUIContractListMiddleware'.

![image](https://user-images.githubusercontent.com/2565797/82137510-8da0c100-9836-11ea-8e40-c66f50b97970.png)

**Comment by StratisIain** [comment]
Hi @ranjancse26  - I assume the above is running on one of the development branches..

If you are connecting to live public chains, can you please utilise the [latest release branch](https://github.com/stratisproject/StratisBitcoinFullNode/tree/release/3.0.8.0).

**Comment by ranjancse26** [comment]
Yes. I was running it based off the development branches as per the Readme instructions mentioned in your repo :) Please fix the readme file if you don't intend the developers to run.
